src/main/java/com/google/firebase/messaging/AndroidNotification.java

```diff
@@ -180,6 +182,15 @@ private AndroidNotification(Builder builder) {
     SECRET,
   }
 
+  private static final Map<Priority, String> PRIORITY_MAP = 
+      ImmutableMap.<Priority, String>builder()
+          .put(Priority.MIN, "PRIORITY_MIN")
```

Comment:
May be override the `toString()` method for `Priority` and put this logic there?

Reply:
Done.